ModularTegustation/fishing/code/fish_market.dm

```diff
@@ -152,3 +157,61 @@
 		if((FISH_RARITY_VERY_RARE + 1) to FISH_RARITY_RARE)
 			fish_worth = 10
 	return round(fish_worth * fish_worth_mod)
+
+
+/obj/machinery/fish_market/proc/check_city()
+	if(SSmaptype.maptype in SSmaptype.citymaps)
```

Comment:
imo, should de-hardcode it
How about making the if() check where the proc is called instead of here, then renaming the proc to something like `udpate_stock()`. That would allow admins to call it manually on any map, and i think that'd be neat

Reply:
It's actually hardcoded because the other god stuff isn't supposed to work outside of those maps.